### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by virag-ky**
In this part of the project, we refactored the code in the main file. We tried out several solutions but this one seemed to work out the best, if you have any suggestions please let us know so we can improve the code further.

Implementation:
* we separated the switch statement from the main file
* we created a refactor file where we put the switch statement
* in the main file we are calling our app to run and we are creating an instance of the App class
* we are using a loop to get the message with the list of options

### CODE REVIEW

File: main [hunk 1]
```diff
@@ -1,58 +1,29 @@
 #!/usr/bin/env ruby
 
-require_relative 'app'
+require_relative 'refactor'
```
Comment (sja-thedude) [inline main:3]: - [ ] Optional: You could just rename your main file to main.rb as its ruby file.

**Review by sja-thedude — APPROVED** [review]
## Project Approved 👍 

Hi @virag-ky & @AsimKhan2019, This is @sja-thedude your code reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Highlights:
- All linters passing
- All requirements completed
- Professional readme

giphy

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main (hunk 1 above)
Comment (virag-ky) [reply main:3]: Okay, thanks 👍🏻

**Comment by AsimKhan2019** [comment]
@sja-thedude Thank you for the approval

**Pull request merged**